packages/platforms/accton/x86-64/as5915-18x/modules/builds/Makefile

```diff
@@ -0,0 +1,6 @@
+KERNELS := onl-kernel-4.14-lts-x86-64-all:amd64
```

Comment:
Please mention in the commit message, what Linux kernel version is used, and also elaborate, why this old version is used and not the current stable release 5.4 or even 5.10.

Reply:
Upgrade kernel to 4.19 which is used by the customer.